conductr_cli/sandbox_run_jvm.py

```diff
@@ -467,6 +468,11 @@ def download_sandbox_image(image_dir, package_name, artefact_type, image_version
     try:
         bintray_auth = bintray_resolver.load_bintray_credentials(raise_error=False)
 
+        if sandbox_version.version_parts(image_version) >= (2, 1, 0):
```

Comment:
`sandbox_version.version_parts` is a "private" function and should not be used from the outside. Can you use `sandbox_version.is_conductr_on_private_bintray` instead: https://github.com/typesafehub/conductr-cli/blob/master/conductr_cli/sandbox_version.py#L6

The general idea is that we encapsulate all the ConductR version checks in `sandbox_version`. The functions that are called from the outside should ask for a certain capability, e.g. is_conductr_on_private_bintray, is_conductr_supportive_of_features, is_sandbox_docker_based.